cfgov/agreements/jinja2/agreements/_database_disclaimer.html

```diff
@@ -1,14 +1,72 @@
 {% import "molecules/social-media.html" as social_media with context %}
 
-<div class="block">
-    <hr>
-</div>
+<div class="block block__padded-top block__border-top">
+    <h2 id="find-issuer">
+        Can't find an issuer?
+    </h2>
+    <p>
+        The drop-down list contains credit card issuers that have
+        submitted credit card agreements as required under
+        Section 204 of the Credit Card Accountability
+        Responsibility and Disclosure Act of 2009 (the CARD Act).
+        If you cannot find a card issuer, it may be that:
+    </p>
+    <ul>
+        <li>
+        The issuer has not yet submitted its agreements
```

Comment:
`<li>` contents could be indented 4 spaces.

Reply:
Fixed in 3a92a97